examples/twittermap/web/public/data/allStatePopulation.json

```diff
@@ -0,0 +1,261 @@
+[
```

Comment:
To reduce the network overhead, I think for these 3 json files, those attributes: `name` in `allStatePopulation.json`; `name`. `stateID`, `stateName` in `allCountyPopulation.json`; and `name`, `countyID`, `countyName`, `stateID`, `stateName` are unnecessary, since all the IDs are unique themselves. And if we get rid of them, we may also save much `TwittermapAppalication` memory (since you load them into memory on the server side).